### PR TITLE
[RFC] remove duplicated explanation

### PR DESCRIPTION
**Pull request by DQNEO**
First, thank you very much for your great effort to maintain this documentation.

I am one of Symfony newbies, and I felt reading documentation is a little bit difficult.
One of the reasons is that there is some duplication of explanations.

I know this PR is a kind of controversial one, but removing duplication would make less confusion for new comers.

---

| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | Symfony 2.7 |
| Fixed tickets | none |

---

### CODE REVIEW

File: quick_tour/the_big_picture.rst [hunk 1]
```diff
@@ -8,123 +8,14 @@ quickly by showing you a simple project in action.
...
-
-When you are finished working on your Symfony application, you can stop
-the server with the ``server:stop`` command:
-
-.. code-block:: bash
-
-    $ php app/console server:stop
-
-.. tip::
+See the Installtion Manual. :doc:`/book/installtion`.
```
Comment (javiereguiluz) [inline quick_tour/the_big_picture.rst]: Minor typo: :doc:`/book/installtion` --> :doc:`/book/installation`

**Comment by DQNEO** [comment]
@jackdelin thank you, fixed it.

**Comment by javiereguiluz** [comment]
@DQNEO I love this pull request. This "deduplication" is really the right thing to do and it will simplify the work of our doc managers. I've created a new pull request (#5497) to reword the text that explains where to look for the Symfony installation help. Of course I took your original commits from this pull request, so you won't lose the credit of your work. Thanks!

**Comment by xabbuh** [comment]
@DQNEO @javiereguiluz Do you suggest to close here in favor of #5497 or do you like to discuss both things separately?

**Comment by DQNEO** [comment]
I closed this one.

**Comment by DQNEO** [comment]
We made it!
d043b626e86b0410e376639ba31f2772e6d578e4

**Pull request closed without merging**